packages/table/src/Centered.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+
+const displayFlex = css`
+  display: flex;
+  flex-grow: 1;
+`;
+
+const centeredHorizontal = css`
+  justify-content: center;
+`;
+
+const centeredVertical = css`
+  align-items: center;
+`;
+
+interface CenteredProps {
+  children: React.ReactNode;
+  horizontal?: boolean;
+  vertical?: boolean;
+}
+
+function Centered({
```

Comment:
@DesignerDave I think you're opposed to this as an export but would be interested in making a utils package for us to consume this internally :)

Reply:
cc @lazytype